package/src/dependencies/SkiaProxy.ts

```diff
@@ -0,0 +1,12 @@
+import type * as Skia from '@shopify/react-native-skia'
+import { createModuleProxy } from './ModuleProxy'
+type TSkia = typeof Skia
+
+/**
+ * A proxy object that lazy-imports @shopify/react-native-skia as soon as the
+ * caller tries to access a property on {@linkcode SkiaProxy}.
+ *
+ * If @shopify/react-native-skia is not installed, accessing anything on
+ * {@linkcode SkiaProxy} will throw.
+ */
+export const SkiaProxy = createModuleProxy<TSkia>('@shopify/react-native-skia', () => require('@shopify/react-native-skia'))
```

Comment:
omg this is cool, much better than we came up with for our lose Reanimated coupling.

Reply:
Haha yea thanks I was also pretty excited when I figured out I could just use Proxies for that. I was a bit worried that Hermes force-included those `require`s but it works!